doc/user/content/release-notes.md

```diff
@@ -48,6 +48,10 @@ Wrap your release notes at the 80 character mark.
 
 {{% version-header v0.7.1 %}}
 
+- **Breaking change.** Remove `CREATE SINK ... AS OF`, which did not have
+  sensible semantics after Materialize restarted. The intent is to reintroduce
```

Comment:
maybe "behavior" rather than "semantics"? The semantics seem fine to me, but we didn't implement them.

Reply:
Sure, done.